### PR TITLE
install/users: document cli --confirm parameter for user creation

### PR DESCRIPTION
**Pull request by J4bbi**
### Description

This documents a simpler way for users following the installation instructions to create activated & confirmed users.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [x] I've added relevant test cases.
- [x] I've added relevant documentation.
- [x] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [x] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [x] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

File: docs/install/index.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ The guide is intended for system administrators and developers who want to try,
...
+This guide covers how to install InvenioRDM locally on your machine, how to set up and configure your system for InvenioRDM.
 
 ## Quick start
 
 #### [1. Install CLI tool](cli.md)
 
 Install the InvenioRDM CLI tool (see [reference](../reference/cli.md)), e.g. via [`pip`](https://pip.pypa.io/en/stable/):
 
-```console
+```bash
```
Comment (fenekku) [inline docs/install/index.md:17]: :heart: :eyeglasses:

File: docs/customize/vocabularies/users.md [hunk 2]
```diff
@@ -73,8 +73,15 @@ db.session.commit()
 
 ## Confirm user
 
-Only confirmed accounts can be logged in. To confirm an account programmatically, create a new shell using `pipenv run invenio shell` and
-run:
+Only confirmed accounts can be logged in. There are two ways to confirm an account as an administrator:
+
+When creating the account pass the `--confirm` parameter like so:
+```shell
```
Comment (fenekku) [inline docs/customize/vocabularies/users.md:79]: Let's pick `bash` or `shell` for all appropriate. I want to say `shell`: it gives just as good contrast and might save us from "bashisms", but I don't know / not a big deal as long as it's just the same one.

File: docs/install/index.md [hunk 3]
```diff
@@ -49,8 +49,9 @@ You will be asked several questions. If in doubt, choose the default.
 
 #### [4. Build, setup and run](build-setup-run.md)
 
-Now the scaffoling is complete, it is time to check the development requirements
-```console
+Now the scaffolding is complete, it is time to check the development requirements
```
Comment (fenekku) [inline docs/install/index.md:52]:
```suggestion
Now that the scaffolding is complete, it is time to check the development requirements
```

File: docs/install/index.md [hunk 4]
```diff
@@ -98,16 +99,23 @@ Depending on whether you are in a local or containerized setup, take note of the
...
 
 **Containerized application**   
 In a fully containerized context, connect to a container first e.g. the web-api container: `docker exec -it my-site-web-api-1 /bin/bash`. Then run the commands from within the container as-is. 
 
-**Steps**
-1- `invenio users create <EMAIL> --password <PASSWORD>`
-2- Activate it with `invenio users activate <EMAIL>`
-3- If you have email verification enabled (as per defaults), you may want to [verifiy the account manually](https://inveniordm.docs.cern.ch/customize/vocabularies/users/#confirm-user) using the user's <EMAIL>.
-4- Allow the user to access the administration panel: `invenio access allow administration-access user <EMAIL>`
+The following command creates an activated and confirmed user (assuming you have email verification enabled as is the default).
```
Comment (fenekku) [inline docs/install/index.md:107]:
```suggestion
**Steps**

The following command creates an activated and confirmed user (assuming you have email verification enabled as is the default). 
```

**Review by fenekku — APPROVED** [review]
Thanks for this! Can be merged right away after minor comments are addressed.

**Comment by J4bbi** [comment]
Thanks for your review @fenekku , I believe I've addressed those issues

**Pull request merged**